helms/odahu-flow-authorization/files/openpolicyagent/policy.rego

```diff
@@ -0,0 +1,65 @@
+package odahu
```

Comment:
IMHO, this is the wrong place for a policy file.
I thought that we would keep policies in the odahu-automation or legion-profiles repository.
I don't want to change the helm chart whenever my policies are changed.